src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -621,7 +606,7 @@ private static bool TryDeserializeJson<TResult>(string json, out TResult result)
         /// <summary>
         /// Performs required static initialization in the Edge context.
         /// </summary>
-        private static void Initialize()
+        private static async Task Initialize()
```

Comment:
Yeah send me those docs, I've wondered which to prefer and hear the compiler generates a lot more code for async/await than return Task so that was my default if I could :)